l10n_it_fatturapa_out/models/attachment.py

```diff
@@ -79,6 +79,17 @@ def write(self, vals):
                 )
         return res
 
+    @api.multi
+    def unlink(self):
+        for attachment_out in self:
+            for invoice in attachment_out.out_invoice_ids:
+                invoice_attachments = invoice.fatturapa_doc_attachments
+                if any([ia.is_pdf_invoice_print
+                        for ia in invoice_attachments]):
+                    invoice.write(
+                        {'fatturapa_doc_attachments': [(5, )]})
```

Comment:
Però così elimini tutti gli allegati in `fatturapa_doc_attachments`, anche quelli eventualmente aggiunti dall'utente

Reply:
Non trovo nel codice che vengano allegati report diversi da quello di default, v. [link](https://github.com/OCA/l10n-italy/blob/bb8ad2bdf64d2eed664660c69b8acb9a3b6efec2/l10n_it_fatturapa_out/wizard/wizard_export_fatturapa.py#L876-L917)

Reply:
Mi riferivo a documenti inseriti manualmente dall'utente nel campo `fatturapa_doc_attachments`

Reply:
quindi dovrei filtrare il report che viene creato di default?

Reply:
sì, con `is_pdf_invoice_print`

Reply:
fatto (qualche tempo fa ;D )